karma.conf.js

```diff
@@ -67,7 +76,7 @@ module.exports = function (config) {
     // test results reporter to use
     // possible values: 'dots', 'progress'
     // available reporters: https://npmjs.org/browse/keyword/karma-reporter
-    reporters: ['dots', 'coverage-istanbul'],
+    reporters: ['progress', 'coverage-istanbul'],
```

Comment:
When running the tests `dots` are not telling us anything. `progress` seems to me as a much better option.